flecsi/util/annotation.hh

```diff
@@ -67,7 +67,10 @@ template<class T>
 struct execute_task : region<execution> {
   /// Set code region name for regions inheriting from execute_task with the
   /// following prefix.
-  inline static const std::string name{"execute_task->" + T::tag};
+  static std::string const & name() {
```

Comment:
@hkaiser 

Can you write a comment on why this change is needed. If this change is justified, please also update other client codes to use this API.

Reply:
@ollielo This change is a workaround for a compiler problem I'm seeing. What other client codes do I have to modify?